code/code.py

```diff
@@ -12,27 +12,25 @@
 "talon": "talon",
 }
 
-regex_ext = re.compile("\.(\S*)\s*")
- 
+# The [^\\/] is specifically to avoid matching something like a .talon folder
+# that would otherwise cause the .talon file action to load
+regex_ext = re.compile("[^\\/]\.(\S*)\s*")
```

Comment:
I think this needs a minor tweak to ensure the correct platform-specific path separator is used, particularly for Windows

`regex_ext = re.compile("[^\\{}]\.(\S*)\s*".format(os.path.sep))`

could you give that a whirl on Linux?

Reply:
I was able to poke at this a bit more today. A windows path such as

C:\...\.talontest\test.talon

will result in this match:

talontest\test.talon

when running talon. This is in spite of the proposed regex doing the correct thing on the online regex testers I typically use.

Escaping the '/' character looks like it fixes it, so i'll merge this in and make that minor tweak.

regex_ext = re.compile("[^\\\/]\.(\S*)\s*")